packages/mongodb-artifact-generator/src/chat/makeSummarizer.test.ts

```diff
@@ -0,0 +1,89 @@
+import {
+  AzureKeyCredential,
+  CORE_ENV_VARS,
+  OpenAIClient,
+  assertEnvVars,
+} from "mongodb-rag-core";
+import { Summary, makeSummarizer } from "./makeSummarizer";
+import { PromptExamplePair } from "./utils";
+
+const { OPENAI_CHAT_COMPLETION_DEPLOYMENT, OPENAI_ENDPOINT, OPENAI_API_KEY } =
+  assertEnvVars(CORE_ENV_VARS);
+
+const openAiClient = new OpenAIClient(
+  OPENAI_ENDPOINT,
+  new AzureKeyCredential(OPENAI_API_KEY)
+);
+
+const openAi = {
+  client: openAiClient,
+  deployment: OPENAI_CHAT_COMPLETION_DEPLOYMENT,
+};
+
+// TODO: Convert this into a .eval file
```

Comment:
can you do as part of this PR or make a follow up ticket?

if doing follow up, it looks like there's some commented out stuff that could be removed

Reply:
thoughts on this @nlarew?

Reply:
I have this partially implemented, going to open a separate PR for it related to https://jira.mongodb.org/browse/EAI-519 and https://jira.mongodb.org/browse/EAI-579